docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -83,6 +83,31 @@ Modeling Measles in GBD 2017
 
    Make sure to check on measles sequelae as well. Our models so far have not paid much attention to the nonfatal side, but it looks like some of the complications can persist well after someone recovers from measles, so maybe that's important to think about.
 
+GBD Hierarchy
+-------------
+
+**Hierarchy table:**
+
++----------------------------------+------------+-----------+
```

Comment:
For simple tables like this, I highly suggest using list-table http://docutils.sourceforge.net/docs/ref/rst/directives.html#list-table

Reply:
I will modify it the table to list.